modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/QuerySpecTests.java

```diff
@@ -63,7 +70,58 @@ public static void afterClass() throws Exception {
         searchRequestParsers = null;
     }
 
-    // TODO add some sort of roundtrip testing like we have now for queries?
+    public static RatedRequest createTestItem(List<String> indices, List<String> types) {
+        String specId = randomAsciiOfLength(50);
+
+        SearchSourceBuilder testRequest = new SearchSourceBuilder();
+        testRequest.size(23);
```

Comment:
Switched to random. Thanks for catching this.

And for the specific number: When using magic numbers in tests I usually choose 23 or 42 or something else that seems unreasonable so whoever reads it thinks about whether this should be something else.